### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by daraosn** [daraosn/Cordova-CanvasCamera#3]
For anyone using this: There's a memory leak in this plugin (you can see with the Profiler). If someone wants to address this, go for it :)

**Comment by binhdocco** [comment]
yup, the app crashes after running about 5 minutes. Could anyone fix it ?

**Comment by mcretien** [comment]
Any idea about how to fix it?
